l293d/driver.py

```diff
@@ -138,17 +39,12 @@ def get_pin_numbering(cls):
 
 # Set GPIO mode
 if not Config.test_mode:
-    if Config.pin_numbering == 'BOARD':
-        v_print('Setting GPIO mode: BOARD')
-        GPIO.setmode(GPIO.BOARD)
-    elif Config.pin_numbering == 'BCM':
-        v_print('Setting GPIO mode: BCM')
-        GPIO.setmode(GPIO.BCM)
-    else:
-        print("pin_numbering must be either 'BOARD' or 'BCM'.")
-        raise ValueError("pin_numbering must be either 'BOARD' or 'BCM'.")
+    pin_num = Config.pin_numbering
```

Comment:
I've removed the checks here as the `Config` class won't allow `pin_numbering` to be anything other than "BOARD" or "BCM".